stable/fluent-bit/templates/config.yaml

```diff
@@ -21,7 +21,7 @@ data:
 {{- if .Values.metrics.enabled }}
         HTTP_Server  On
         HTTP_Listen  0.0.0.0
-        HTTP_Port    2020
+        HTTP_Port    {{ .Values.metrics.service.port | default "2020" }}
```

Comment:
values are the defaults. You can skip adding an | default.

Reply:
Good point :)